app/src/main/java/com/appunite/loudius/domain/store/UserLocalDataSource.kt

```diff
@@ -26,6 +26,7 @@ import javax.inject.Singleton
 @Singleton
 class UserLocalDataSource @Inject constructor(@ApplicationContext context: Context) {
 
+    // suggestion: it would be nice if would use some encrypted credentials store
     companion object {
         private const val FILE_NAME = "com.appunite.loudius.sharedPreferences"
         private const val KEY_ACCESS_TOKEN = "access_token"
```

Comment:
That's a good idea, we thought about it too. We have a task for that in the backlog: 
https://loudius.atlassian.net/jira/software/projects/LD/boards/3/backlog?selectedIssue=LD-24

Reply:
The task exists - so I resolve the issue.